docker/set_env.sh

```diff
@@ -104,7 +104,7 @@ else
   die "Unknown environment"
 fi
 
-add_to_docker_opts CXXFLAGS="${CXXFLAGS} -Wall ${WERROR:+-Werror}"
+add_to_docker_opts CXXFLAGS="${CXXFLAGS} -Wall -Wextra -Wpedantic -Wshadow -Wno-conversion -Wno-unused-parameter -Werror -Wno-deprecated-copy ${WERROR:+-Werror}"
```

Comment:
@JensWehner I left `-Wconversion` and `-Wunused-parameter` for you.

Reply:
what do you mean?

Reply:
I couldn't fix all the warnings generated by `-Wconversion` in xtp hence there is still an `-Wno-conversion` here. Same for `-Wunused-parameter`.